packages/web-components/src/divider/divider.ts

```diff
@@ -12,25 +12,25 @@ import { BaseDivider } from './divider.base.js';
  */
 export class Divider extends BaseDivider {
   /**
+   * Determines the alignment of the content within the divider. Select from start or end. When not specified, the content will be aligned to the center.
    * @public
    * @remarks
-   * Determines the alignment of the content within the divider. Select from start or end. When not specified, the content will be aligned to the center.
    */
   @attr({ attribute: 'align-content' })
   public alignContent?: DividerAlignContent;
 
   /**
+   * A divider can have one of the preset appearances. Select from strong, brand, subtle. When not specified, the divider has its default appearance.
    * @public
    * @remarks
-   * A divider can have one of the preset appearances. Select from strong, brand, subtle. When not specified, the divider has its default appearance.
    */
```

Comment:
This block was incorrect in its original form, but the `@remarks` section is now empty. We usually provide info about the HTML attribute in these cases:
 
```ts
  /**
   * Indicates the styled appearance of the divider.
   * 
   * @public
   * @remarks
   * HTML Attribute: `appearance`
   */
```

I also removed the specific content callouts and defaults, since the values can change between implementations.